tests/lib/systemd.sh

```diff
@@ -68,9 +68,10 @@ systemd_stop_units() {
             echo "Ensure the service is active before stopping it"
             retries=20
             systemctl status "$unit" || true
-            while systemctl status "$unit" | grep "Active: activating"; do
+            while systemctl status "$unit" | grep -q "Active: activating"; do
```

Comment:
Hm this should probably do `systemctl show -p ActiveState "$unit" | grep -q 'ActiveState=activating'` or
 ```
while [[ "$(systemctl is-active "$unit"" = "activating" ]]; then
...
done
```
Either way, the proposed fix works too.

Reply:
Yeah, there's some simplification possible but for the sake of not checking Ubuntu 14.04 I went for the simplest thing possible.